python intro/calculator/calc.py

```diff
@@ -6,11 +6,11 @@ def mainFunc ():
     var2 = input("Input Second Number: ")
     calcType = input("What do you want to calculate for? ")
 
-    if calcType != "sum" and calcType != "product" and calcType != "difference" and calcType != "quotient":
+    if calcType not in ["sum", "product", "difference", "quotient"]:
         print("Try using keywords; sum, product, difference or quotient.")
-        success = 0
+        success = False
```

Comment:
You were using these as booleans so I figured they should be booleans .

Additionally, you could simplify this if statement. I'll leave that as an exercise though

Reply:
Ah I see, for some reason I was thinking it had to be 0 and 1, rather than True and False. I was baked so that's probably why.